diff --git a/shared/sdl/sdl_window.cpp b/shared/sdl/sdl_window.cpp
index 18daa067..c3e94ccc 100644
--- a/shared/sdl/sdl_window.cpp
+++ b/shared/sdl/sdl_window.cpp
@@ -83,7 +83,16 @@ const vidmode_t r_vidModes[] = {
     { "Mode  9: 1600x1200",		1600,	1200 },
     { "Mode 10: 2048x1536",		2048,	1536 },
     { "Mode 11: 856x480 (wide)", 856,	 480 },
-    { "Mode 12: 2400x600(surround)",2400,600 }
+    { "Mode 12: 2400x600(surround)",2400,600 },
+    { "Mode 13: 1280x720 (16:9)",	1280,	720 },
+    { "Mode 14: 1600x900 (16:9)",	1600,	900 },
+    { "Mode 15: 1920x1080 (16:9)",	1920,	1080 },
+    { "Mode 16: 2560x1080 (21:9)",	2560,	1080 },
+    { "Mode 17: 2560x1440 QHD (16:9)",	2560,	1440 },
+    { "Mode 18: 3440x1440 (21:9)",	3440,	1440 },
+    { "Mode 19: 3840x1600 (24:10)",	3840,	1600 },
+    { "Mode 20: 3840x2160 4K (16:9)",	3840,	2160 },
+    { "Mode 21: 5120x1440 (32:9)",	5120,	1440 }
 };
 static const int	s_numVidModes = ARRAY_LEN( r_vidModes );
 
